.github/actions/run-and-record-tests/action.yml

```diff
@@ -62,7 +62,7 @@ runs:
 
 
     - name: Commit and push recordings
-      if: ${{ inputs.inference-mode == 'record' }}
+      if: ${{ inputs.inference-mode == 'record' ||  inputs.inference-mode == 'record-if-missing' }}
```

Comment:
good catch but yes please make this an independent PR because stuff (larger PRs) can also get reverted so it is best when independently useful things are landed separately.

Reply:
sure, let me open a PR for this.